adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -849,6 +1119,14 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     DatabaseVariantAnnotationRDD(rdd, sd)
   }
 
+  /**
+   * Loads NucleotideContigFragments stored in Parquet, with metadata.
+   *
```

Comment:
Something is wonky with the github web UI, look at line 1136 in view whole file, it should be "Load VariantAnnotations ...".  Then line 1126 def loadParquetVariantAnnotations is missing doc.

https://github.com/fnothaft/adam/blob/27b769c69e7a2a864ba7a99d824213df72909acc/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala#L1136